packages/flutter/lib/src/widgets/animated_list.dart

```diff
@@ -333,7 +332,31 @@ class AnimatedListState extends State<AnimatedList> with TickerProviderStateMixi
         ..sort();
     });
 
-    controller.reverse().then<void>((void value) {
+    if (builder != null) {
+      controller.reverse().then<void>((void value) {
+        _removeActiveItemAt(_outgoingItems, outgoingItem.itemIndex).controller.dispose();
+
+        // Decrement the incoming and outgoing item indices to account
+        // for the removal.
+        for (_ActiveItem item in _incomingItems) {
+          if (item.itemIndex > outgoingItem.itemIndex)
+            item.itemIndex -= 1;
+        }
+        for (_ActiveItem item in _outgoingItems) {
+          if (item.itemIndex > outgoingItem.itemIndex)
+            item.itemIndex -= 1;
+        }
+
+        setState(() {
+          _itemsCount -= 1;
+        });
+      });
+    }
+    /// [5/24/2019]
```

Comment:
What is this date breadcrumb here for? What does it link to?

Reply:
@Piinks

The date and part I edited.

You can remove the comment.

Sorry.

Reply:
Can you remove this comment?

Reply:
@goderbauer Done.